src/app/containers/ATIAnalytics/canonical/index.jsx

```diff
@@ -11,9 +12,11 @@ const CanonicalATIAnalytics = ({ pageviewParams }) => {
   }, [atiPageViewUrl]);
 
   return (
-    <noscript>
-      <img height="1px" width="1px" alt="" src={atiPageViewUrl} />
-    </noscript>
+    <noscript
```

Comment:
Definitely comment explaining why this is necessary, best to include link to the relevant issue or PR in react

Reply:
Repeat comments in all places you've done this

Reply:
Done